com.unity.netcode.gameobjects/Runtime/AssemblyInfo.cs

```diff
@@ -12,3 +12,4 @@
 [assembly: InternalsVisibleTo("Unity.Netcode.RuntimeTests")]
 [assembly: InternalsVisibleTo("Unity.Netcode.TestHelpers.Runtime")]
 [assembly: InternalsVisibleTo("Unity.Netcode.Adapter.UTP")]
+[assembly: InternalsVisibleTo("Unity.Multiplayer.Tools.Adapters.Ngo1WithUtp2")]
```

Comment:
What is this here for?

Reply:
This is required so that we can access the internals of NGO, and more specifically the `UnityTransport`. We need this to access to the Network Driver, and register callbacks for our adapters to know when the transport is initialized.

Reply:
That's right, I forgot we moved UTP into this assembly from the old one (on line 14 here) :) disregard